### PR TITLE
Various renames to normalise namespaces

### LINKED ISSUE #387: Double check namespaces
Check for plural vs. non-plural - namespaces should be singular, e.g `fc.chart` as opposed to `fc.charts`

### PR DESCRIPTION
**Pull request by chrisprice**
Fixes #387.

I'm bound to have missed some but hopefully this catches most of them!

### CODE REVIEW

**Comment by chrisprice** [comment]
Annotation namespace changes in, should be GTG.

File: _SpecRunner.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<!doctype html>
```
Comment (ColinEberhardt) [inline _SpecRunner.html]: we should probably add _SpecRunner to .gitignore ?

**Comment by ColinEberhardt** [comment]
One minor comment, but other than that :+1:

**Comment by chrisprice** [comment]
Good spot, merging...

**Pull request merged**